src/Traits/HasPermissions.php

```diff
@@ -7,20 +7,44 @@
 trait HasPermissions
 {
     /**
-     * Grant the given permission to a role.
+     * Grant the given permission(s) to a role.
      *
-     * @param  $permission
+     * @param string|array|Permission|\Illuminate\Support\Collection $permissions
+     * @param array $params
      *
      * @return HasPermissions
      */
-    public function givePermissionTo($permission)
+    public function givePermissionTo($permissions, ...$params)
     {
```

Comment:
Create an new exception for this in the Exceptions dir. I'll rename it myself if I can improve on it after pulling.